tests/lib/rules/alphabetical-paths.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @fileoverview Tests for `alphabetical-paths` rule
+ * @author Ondřej Brejla <ondrej@brejla.cz>
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var RuleTester = require("eslint").RuleTester,
+    fixtures = require("../../fixtures"),
+    rule = require("../../../lib/rules/alphabetical-paths");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+var ERROR = {
+    message: "Required paths are not in alphabetical order.",
+    type: "CallExpression"
+};
+
+var ruleTester = new RuleTester();
+
+ruleTester.run("alphabetical-paths", rule, {
+
+    valid: [
+        fixtures.ALPHABETICAL_PATHS_NO_PATH,
+        fixtures.ALPHABETICAL_PATHS_NO_PATH_EMPTY_ARRAY,
+        fixtures.ALPHABETICAL_PATHS_ONE_PATH_IN_ARRAY,
+        fixtures.ALPHABETICAL_PATHS_MORE_PATHS_IN_ARRAY,
+
+        fixtures.ALPHABETICAL_PATHS_REQUIRE_NO_PATH,
+        fixtures.ALPHABETICAL_PATHS_REQUIRE_NO_PATH_EMPTY_ARRAY,
+        fixtures.ALPHABETICAL_PATHS_REQUIRE_ONE_PATH_IN_ARRAY,
+        fixtures.ALPHABETICAL_PATHS_REQUIRE_MORE_PATHS_IN_ARRAY,
+
+        {
+            code: fixtures.ALPHABETICAL_PATHS_BASENAME_VALID_ORDER,
+            options: [ "basename" ]
+        }
+    ],
```

Comment:
Can you scour through the existing fixtures and see what *non-AMD* fixtures you can add? It will be great to see if the CommonJS and other flavors don't result in an error with this rule. (And if they do, you might need to add checks like `util.isAmdDefine()`.